src/foam/u2/view/RichChoiceView.js

```diff
@@ -511,7 +518,8 @@ foam.CLASS({
                   .add(self.FILTER_.clone().copyFrom({ view: {
                     class: 'foam.u2.view.TextField',
                     placeholder: this.searchPlaceholder || 'Search... ',
-                    onKey: true
+                    onKey: true,
+                    view: self.inputView
```

Comment:
TextField already extends input, cant we just set autofocus to true on here?

Reply:
We can, but I didn't find the easy way to update the focus when dropdown is showing rendered view with input. So self.inoutView is for this purpose